### PR TITLE
chore: bump nMinimumChainWork, defaultAssumeValid and m_assumed_blockchain_size for mainnet and testnet

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -476,18 +476,18 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_DIP0024].nFalloffCoeff = 5;      // this corresponds to 10 periods
 
         // The best chain should have at least this much work.
-        consensus.nMinimumChainWork = uint256S("0x000000000000000000000000000000000000000000000000022f14ac5d56b5ef"); // 470000
+        consensus.nMinimumChainWork = uint256S("0x000000000000000000000000000000000000000000000000027babc01f54b721"); // 771500
 
         // By default assume that the signatures in ancestors of this block are valid.
-        consensus.defaultAssumeValid = uint256S("0x0000009303aeadf8cf3812f5c869691dbd4cb118ad20e9bf553be434bafe6a52"); // 470000
+        consensus.defaultAssumeValid = uint256S("0x0000096633fa151a8112e499a16cc86007f8b57e3e70e4614fc4fdf11eeb22b6"); // 771500
```
Comment (UdjinM6) [inline src/chainparams.cpp:482]:
```
> dash-cli getblock 0000096633fa151a8112e499a16cc86007f8b57e3e70e4614fc4fdf11eeb22b6
error code: -5
error message:
Block not found
> dash-cli getblockhash 771500
00000067b30c1e082086bf9da8cf49344b90d8fda37b9051875c4c1420549b6f
> dash-cli getblock 00000067b30c1e082086bf9da8cf49344b90d8fda37b9051875c4c1420549b6f
{
...
  "chainwork": "000000000000000000000000000000000000000000000000027babbb45da9fa4",
..
}
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
pls see below

File: src/chainparams.cpp (hunk 1 above)
Comment (PastaPastaPasta) [reply src/chainparams.cpp:482]: Hmmm
```
pasta@pastas-MacBook-Pro ~/workspace/dash                                                                                             [14:35:14]
> $ ./src/dash-cli --testnet getblock 0000096633fa151a8112e499a16cc86007f8b57e3e70e4614fc4fdf11eeb22b6                            [±develop ●●▴]
{
  "hash": "0000096633fa151a8112e499a16cc86007f8b57e3e70e4614fc4fdf11eeb22b6",
  "confirmations": 38,
  "size": 277,
  "height": 771500,
  "version": 536870912,
  "versionHex": "20000000",
  "merkleroot": "a63563f4c8111584960add5969cb487a5ef6ef923db28877e42f6994531b5509",
  "tx": [
    "a63563f4c8111584960add5969cb487a5ef6ef923db28877e42f6994531b5509"
  ],
  "cbTx": {
    "version": 2,
    "height": 771500,
    "merkleRootMNList": "e943367c5fb76a0990fa492ba71d947da260a9a1dc4c81068cc3f34eab252edb",
    "merkleRootQuorums": "c9e43d55e6ac2329799c8d3a4afe5a05314491ec58c99bcc084ffccc99771c01"
  },
  "time": 1659309604,
  "mediantime": 1659306332,
  "nonce": 223423,
  "bits": "1e0fffff",
  "difficulty": 0.0002441371325370145,
  "chainwork": "000000000000000000000000000000000000000000000000027babc01f54b721",
  "nTx": 1,
  "previousblockhash": "00000a99dfbcd8d96976db2ffcb417d956d09f4ba021de98d5fb18a7cc850159",
  "nextblockhash": "000006354a728a1442c35b7c9620d7dc92a6c7d0e2787a684cdaf41b4d388be0",
  "chainlock": false
}
```

Comment (PastaPastaPasta) [reply src/chainparams.cpp:482]: I was on a fork / had invalidated the tip it looks like... I guess I need to redo most of this testnet stuff

Comment (UdjinM6) [reply src/chainparams.cpp:482]:
```
> dash-cli getbestchainlock
{
  "blockhash": "000000f695b071fa90f5ff552261125e3598983af6975aa537ea9bb7820f4042",
  "height": 777559,
  "signature": "87fab7154dab027add1c5b8baad00e027cf1811b14d51a4c320853a9b4ac98fe8f58c659311661fa82b6ed2332c0db87132a7a5c664d0124a5e395b4c5b8b8fc872a4488708697d21ce936508d6d058c6ab3f331a6673bdedd49eccaf87b26a4",
  "known_block": true
}
```

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK

**Pull request merged**